.gitignore

```diff
@@ -1,14 +1,7 @@
 .idea/
 .vagrant/
 .vscode/
-/microshift
-/paack-result/
 Vagrantfile
```

Comment:
Don't we need to keep /microshift as the exe still goes to the project root dir, right?